docs/topics/maven.md

```diff
@@ -23,6 +23,27 @@ To use JDK 17, in your `.mvn/jvm.config` file, add:
 --add-opens=java.base/java.io=ALL-UNNAMED
 ```
 
+## Declare repositories
+
+To access artifacts in repositories that can be consumed by Maven builds, specify the ID and URL of each repository in
+the `<repositories>` tag:
+
+```xml
+<repositories>
+    <repository>
+        <id>mavenCentral</id>
+        <url>https://repo1.maven.org/maven2/</url>
+    </repository>
+</repositories>
+```
+
+> We recommend that you avoid adding `mavenLocal()` as a repository. If you declare `mavenLocal()` as a repository then you
```

Comment:
This feels confusing, being `mavenLocal()` the Gradle syntax.  
Plus the local repository is used by default by Maven, so no need to declare it.

Reply:
Yeah this one too. I probably picked a commit by accident instead of the entire PR. :eyes: